_posts/2020-07-08-Book-Five-Dysfunctions.md

```diff
@@ -0,0 +1,173 @@
+There are multiple ways of drawing up a broad profile of the members in your team. 
+The Myers-Briggs Type Indicator (MBTI) is a stock standard way of charting this, categorizing people on the four principal psychological functions (sensation, intuition, feeling, and thinking)  which gives us 16 distinct personality types, each with their typical strengths and weaknesses. 
+Other common profiling tools are the Business Insights analysis, Everything DiSC or even the Interpersonal Circumplex (also known as the Rose of Leary). 
+
+{:refdef: style="text-align: center;"}
+<img src="{{ '/img/2020-07-08-Book-Five-Dysfunctions/mbti.png' | prepend: site.baseurl }}" alt="Workspace" class="image" style="margin:0px auto; max-width:100%">
+{: refdef}
+{:refdef: style="text-align: center;"}
+Myers-Briggs Type Indicator Personality Types (taken from [Adioma blogpost](https://blog.adioma.com/16-personality-types/){:target="_blank" rel="noopener noreferrer"})
+{: refdef}
+
+When there are clear key figures within the team, they can further this trust through demonstrating vulnerability first. 
+When these key figures of the team are open about their own weaknesses but also strengths, this behavior will become more natural and acceptable to other members of the team that might not yet have had experience with this openness. 
+When considering George Kohlrieser’s Secure Base Leadership paradigm, he would phrase it as such: A Secure Base Leader builds trust, delivers change and inspires the focus that people need to actively engage in their work and create conditions for innovation. 
+Secure Base Leaders generate high performance by acting as a “secure base” for the people they lead.
+
+##	Fear of Conflict
+When there is no trust, there is no fertile ground for discussion within the group that may result in open conflict as understandings and opinions might differ. 
+These discussions are necessary to come to a mutual conclusion in how to approach and resolve the vision set forth by the group. 
+This discussion gives a platform for every member of the group to be heard and increases buy-in drastically. 
+The idea is similar to the thesis-antithesis-synthesis concept of Johann Gottlieb Fichte, the German philosopher, who postulated such a progression of ideas to come to the best solution for any problem or proposition.
+
+The meeting culture in most companies is a very good example of this. 
+They are mostly led by the organizer trying to convey information of his point of view in an almost ex cathedra capacity, with very little interactions between those attending the meeting. 
+There will some vocal members, but the majority will sit there accepting what is being told at face value. 
+While some meetings are all about disseminating information, most of them are organized with the idea of getting to a consensus about certain topics. 
```

Comment:
disseminating is not easily understandable.
Is it possible to vulgarise this?

Reply:
Same remark as before